ipn/ipnlocal/serve.go

```diff
@@ -498,6 +499,7 @@ func (b *LocalBackend) addTailscaleIdentityHeaders(r *httputil.ProxyRequest) {
 	}
 	r.Out.Header.Set("Tailscale-User-Login", user.LoginName)
 	r.Out.Header.Set("Tailscale-User-Name", user.DisplayName)
+	r.Out.Header.Set("Tailscale-Headers-Info", "https://tailscale.com/serve-headers")
```

Comment:
should this be a /s/ URL?  I'm assuming this will redirect somewhere?

Reply:
Oh yes! Didn't realize we had that convention, updating